src/algorithms/tracking/IterativeVertexFinder.cc

```diff
@@ -207,7 +222,37 @@ std::unique_ptr<edm4eic::VertexCollection> eicrecon::IterativeVertexFinder::prod
          (float)vtx.time(),
     }); // vtxposition
     eicvertex.setPositionError(cov);                          // covariance
-  }
+
+    for (const auto& t : vtx.tracks()) {
+#if Acts_VERSION_MAJOR >= 33
+      const auto& trk = &t.originalParams;
+      const auto& par = finderCfg.extractParameters(trk);
+#else
+      const auto& par = *t.originalParams;
+#endif
+      m_log->trace("Track local position from vertex = {} mm, {} mm", par.localPosition().x() / Acts::UnitConstants::mm, par.localPosition().y() / Acts::UnitConstants::mm);
+      float loc_a = par.localPosition().x();
+      float loc_b = par.localPosition().y();
+
+      for (const auto& part : *reconParticles) {
+        const auto& tracks = part.getTracks();
+        for (const auto trk : tracks) {
+          const auto& traj = trk.getTrajectory();
+          const auto& trkPars = traj.getTrackParameters();
+          for (const auto par : trkPars) {
+            constexpr double acts_to_edm4eic = edm4eic::unit::mm / Acts::UnitConstants::mm;
+            if(fabs(par.getLoc().a - loc_a * acts_to_edm4eic) < 1.e-4 && fabs(par.getLoc().b - loc_b * acts_to_edm4eic) < 1.e-4) {
```

Comment:
Avoid dimensionful magic numbers.

Reply:
Addressed. Let's assume that we don't need to change this value. The algorithm depends on numerical tolerance, and it will be replaced with another approach once that's supported by the EDM.